<launch>
	<arg name="marker_size" default="4.0" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/camera/depth_registered/points" />
	<arg name="cam_info_topic" default="/camera/depth_registered/camera_info" />		
	<arg name="output_frame" default="/camera_link" />

	 <include file="$(find openni2_launch)/launch/openni2.launch">
  		<param name="depth_registration" type="boolean" value="true" />
   		<arg name="publish_tf" value="true" />
  	</include>
  	
	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />

	<node pkg="watermellon" type="wm_object_node" name="wm_object" output="screen">
	</node>

	<node pkg="rviz" type="rviz" output="screen"
		name="rviz"
		args="-d $(find watermellon)/rviz/wm_object_training.rviz"/>


</launch>
